### PR TITLE
Add content at the root of the API server

### LINKED ISSUE #320: api server landing page
continuation to #119  (and may be some other recently closed issue I failed to find ATM)

ATM both https://api.dandiarchive.org/ and https://api.dandiarchive.org/api/ stimply 404. IMHO it is suboptimal. I would have at least populated them with some basic information about API server (URL to swagger, may be this repository) or at least redirected somewhere (https://api.dandiarchive.org/swagger/ might not be the worst choice).

### PR DESCRIPTION
**Pull request by waxlamp**
Closes #320.

### CODE REVIEW

**Review by mvandenburgh — APPROVED** [review]
LGTM, linting errors will need to be fixed before merging.

File: dandiapi/api/views/__init__.py [hunk 1]
```diff
@@ -31,4 +32,5 @@
     'users_search_view',
     'stats_view',
     'info_view',
+    'root_content_view',
```
Comment (yarikoptic) [inline dandiapi/api/views/__init__.py:35]: FWIW, someone eventually should sort all those entries IMHO

**Review by yarikoptic — COMMENTED** [review]
Looks lovely.  Doesn't address 404 of `/api` though. Even if we are expecting it to serve something later on, for now it could probably just provide the same page  as for the `/`?

**Comment by waxlamp** [comment]
> Looks lovely. Doesn't address 404 of `/api` though. Even if we are expecting it to serve something later on, for now it could probably just provide the same page as for the `/`?

I'd rather not put something there--I really don't see what value it would bring, and it would likely confuse any tool that did try to make a request to that location. Right now, there's literally nothing to be found there, so I think it would be harmful to put even friendly HTML there, since that provides a signal to downstreams that they found something valuable. I am not too worried about humans hitting that path either, since we have plenty of resources that are more easily found (including the new content at the root of the Django site) that will guide them to the more appropriate places to go.

**Pull request merged**